specification/validated-relationships-service-api.yaml

```diff
@@ -481,36 +798,259 @@ components:
+                            answer:
+                              type: array
+                              items:
+                                oneOf:
+                                  - type: object
+                                    properties:
+                                      valueString:
+                                        type: array
+                                        description: |
+                                          Each name(s) should be a separate item in the list. The first given name may include multiple names, separated by a space.
+                                          Subsequent names must be broken down into list items. For example, the input `[Jane Marie Anne, Jo Adele]` returns `[Jane Marie Anne, Jo, Adele]`.
+
+                                        maxItems: 5                                  
+                                        example: [Jane Marie Anne]
+                                        items:
+                                          type: string
+                                          maxLength: 35
+                                          example: Jane
+
+                        - type: object
+                          required:
+                            - linkId
+                            - text
+                            - answer
+                          properties:
+                            linkId:
+                              type: string
+                              enum: ["patient_name_family"]
+                            text:
+                              type: string
+                              example: "Family name (often called Surname)."
+                            answer:
+                              type: array
+                              minItems: 1
+                              maxItems: 1
+                              items:
+                                oneOf:
+                                  - type: object
+                                    properties:
+                                      valueString:
+                                        type: string
+                                        example: "Smith"
+
+              - type: object
+                required:
+                  - linkId
+                  - text
+                  - answer
+                properties:
+                  linkId:
+                    type: string
+                    enum: ["patient_birthDate"]
+                  text:
+                    type: string
+                    example: "Date of Birth"
+                  answer:
+                    type: array
+                    minItems: 1
+                    maxItems: 1
+                    items:
+                      oneOf:
+                        - type: object
+                          properties:
+                            valueString:
+                              type: string
```

Comment:
`format: date`

Reply:
a DOB provided in the QuestionnaireResponse will have been provided by a human (we are only capturing human-entered demographics PAS will be validating that a valid date has been provided